cip-0046/cip-0046.md

```diff
@@ -0,0 +1,159 @@
+<pre>
+  CIP: 0046
```

Comment:
Need to double check the number but I suggest we hold off on that until we're ready to make it a CIP in the public repo.